src/TransactionTestsFiller/ttEIP2930/accessListStorage0x0001Filler.json

```diff
@@ -0,0 +1,30 @@
+{
+    "accessListStorage0x0001" :
+    {
+        "expectException" :
+        {
+            "<Berlin" : "TR_TypeNotSupported"
+        },
+        "transaction" :
+        {
+            "accessList" : [
+                    {
+                        "address" : "0xa95e7baea6a6c7c4c2dfeb977efac326af552d87",
+                        "storageKeys" : [
+                            "0x0001"
```

Comment:
Check the rlp here. I think it reads as value thus 00 omited.
Try with bigint prefix and doucle check that rlp also contains 00 on this field

Reply:
Updated to have "0x:bigint". Fails as expected with exception "2930AccessListStorageHashTooShort".